code/game/objects/items/stacks/rods.dm

```diff
@@ -6,7 +6,7 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 /obj/item/stack/rods
 	name = "metal rod"
-	desc = "Some rods. Can be used for building, or something."
+	desc = "Some rods. Can be used for building or something."

```

Comment:
As rock, you dont need to be an armchair nut to fucking know that the , isnt needed I dknt think the description of rods needs to be deep its fucking rods